packages/components/organisms/f-header/CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ Latest (to be added to next version)
 ### Changed
 - Updated version of `f-button`.
 
+### Fixed
```

Comment:
As this is a PI, might be good to make it a new version, so it gets updated asap